tests/test_file_watcher.py

```diff
@@ -0,0 +1,48 @@
+import os
+import shutil
+import tempfile
+import time
+import unittest
+from collections import defaultdict
+
+from watchdog.observers import Observer
+
+from liste_files import FileFinallyCreatedEventHandler
```

Comment:
Typo in `liste_files` file name. IDE didn't help.

Reply:
You are right, totally forgot, rename to `file_event_handler` :heavy_check_mark: